Grammar fixes (#1966)
hacks-guide/Guide_3DS: _pages/en_US/move-emunand.txt

{: .notice--info}

1. Launch GodMode9 by holding (Start) during boot

> 1. If you are prompted to create an essential files backup, press (A) to do so, then press (A) to continue once it has been completed

Reviewer: ```suggestion
1. If you are prompted to create an essential files backup, press (A) to do so, then press (A) to continue once it is complete
```

Reviewer: This was not added.

The game will be outputted to the `/gm9/out/` folder on your SD card with the name `<TitleID>.gbavc.sav`.
{: .notice--info}

> To identify a `<TitleID>.gbavc.sav` file's Title ID, you can get a listing of all games on the system and their corresponding Title IDs by hovering over `[A:] SYSNAND SD`, holding (R), and pressing (A) at the same time, then select "Search for titles".

Reviewer 2: This is uh, wrong. It was fine like it was.

  + This will not overwrite your boot9strap installation
1. Input the key combo given to unlock SysNAND (lvl1) writing
  + This process will take some time

> 1. Once it has been completed, press (A) to continue

Reviewer: ```suggestion
1. Once it is complete, press (A) to continue
```

Reviewer: This was not added.
